kusto/kql/builder.go

```diff
@@ -0,0 +1,115 @@
+package kql
```

Comment:
Now that we are near the end it's time to update the docs with the new method:
- README.md
- sample (see stmt_example_test)
- doc.go
- update CHANGELOG.md (Make a ## [Unreleased] section)